macros/plot_scurves_by_thresh.py

```diff
@@ -1,4 +1,4 @@
-from plot_scurve import *
+from macros.plot_scurve import *
```

Comment:
This script imports `plot_scurve` but doesn't use it, probably remove the `import`?